docs/docs/reference/index.md

```diff
@@ -0,0 +1,21 @@
+# API Reference:
+
+<div class="grid cards" markdown>
+
+- [Inference](inference.md)
+  <br/>
+  *XXX*
```

Comment:
I was not sure of what to put here, if you have suggestion that would be perfect.